README.md

```diff
@@ -192,13 +203,53 @@ Distributed under the MIT License. See `LICENSE.txt` for more information.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- CONTACT -->
-
-## Contact
-
-Michael Marchand - MichaelDavidMarchand@gmail.com
-
-Project Link: [https://github.com/dnvr-zero](https://github.com/dnvr-zero)
+<!-- Contributors -->
```

Comment:
This is a markdown/html table showing our photos, names, and links to github and linkedin